torch/nn/modules/pooling.py

```diff
@@ -532,18 +533,21 @@ class AvgPool2d(_AvgPoolNd):
         >>> input = torch.randn(20, 16, 50, 32)
         >>> output = m(input)
     """
+    __constants__ = ['kernel_size', 'stride', 'padding', 'ceil_mode', 'count_include_pad', 'divisor_override']
```

Comment:
just curious: was this necessary to get a test to pass or something?

Reply:
yes, this is for the jit test